alot/commands/globals.py

```diff
@@ -1174,3 +1174,30 @@ def apply(self, ui):
         # flush index
         if self.flush:
             ui.apply_command(commands.globals.FlushCommand())
+
+
+@registerCommand(
+    MODE, 'confirmsequence',
+    arguments=[
+        (['msg'], {'help': 'Additional message to prompt',
```

Comment:
additional sounds as if it was optional, but it shouldn't be right?

Reply:
It is, though -- there's a defaut message that doesn't require you to give additional information about what's being prompted. But if it should be mandatory, I'd be happy to change.

Reply:
I guess that would make sense, don't you think?

Reply:
The default message is just "Confirm sequence?"; I didn't see the need for making any additional prompt necessary. Maybe the entire prompt should just be passed in as a mandatory argument?